exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/scan/convert/WriterBuilder.java

```diff
@@ -0,0 +1,230 @@
+/*
```

Comment:
I have not found a function to refer to this builder. To avoid developers forgetting this good helper, is it possible to give a sample for use (in unit test?)?

Reply:
Well, that is odd. Maybe this is an earlier draft of something that ended up somewhere else? I'll have to hunt down the answer. I'll do that when I update this PR after doing the smaller PRs that James requested.